dev-libs/websocketpp/patches/websocketpp-0.8.1.patch

```diff
@@ -0,0 +1,30 @@
+From 1d2cfa905f7200749055f8b28a2fe5d442d822e8 Mon Sep 17 00:00:00 2001
+From: David Murphy <murphman@gmail.com>
+Date: Mon, 2 Sep 2019 14:03:10 -0400
+Subject: Websocketpp: CMake configuration updates for port to Haiku
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2786aba..c58ebf7 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -3,6 +3,7 @@
+ # Minimum cmake requirement. We should require a quite recent
+ # cmake for the dependency find macros etc. to be up to date.
+ cmake_minimum_required (VERSION 2.8.8)
++set(CMAKE_TRY_COMPILE_TARGET_TYPE "STATIC_LIBRARY")
```

Comment:
Any reason why this was added? There isn't a static nor shared library (only the cmake files that get installed)
![websocketpp2](https://user-images.githubusercontent.com/16057090/65383066-a1c80900-dd0f-11e9-848d-71628121d126.png)